src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderServiceTest.java

```diff
@@ -92,41 +97,32 @@ public void beforeEach() {
   @Test
   void createSnapshotRequest() {
     UUID datasetId = UUID.randomUUID();
-    String email = "user@gmail.com";
     SnapshotAccessRequestResponse response = new SnapshotAccessRequestResponse();
-    when(snapshotRequestDao.create_old(
-            datasetId, SnapshotBuilderTestData.createSnapshotAccessRequest(), email))
+    when(snapshotRequestDao.create(
+            SnapshotBuilderTestData.createSnapshotAccessRequest(datasetId), TEST_USER.getEmail()))
```

Comment:
Should this be `snapshotId` now?
```suggestion
            SnapshotBuilderTestData.createSnapshotAccessRequest(snapshotId), TEST_USER.getEmail()))
```